src/controller/activity.controller.ts

```diff
@@ -10,6 +10,11 @@ export default class ActivityController extends BaseController {
 
   async getActivities(req: Request, res: Response) {
     const userId = (req as any).user?.id ?? TestUserId;
+
+    if (!userId) {
+      return this.error(res, '--activities/user-not-found', 'User not found', 404);
```

Comment:
this are unrelevant guy, please revert it back

Reply:
Will they spoil the code?